### PR TITLE
This month's spelling fix PR

### PR DESCRIPTION
**Pull request by harmonyn**
I can't help it I have spelling ocd

### CODE REVIEW

**Comment by nicbn** [comment]
i saw that

File: code/game/gamemodes/clock_cult/clock_effects/spatial_gateway.dm [hunk 1]
```diff
@@ -119,7 +119,7 @@
 		qdel(src)
 		return FALSE
 	if(!sender)
-		visible_message("<span class='warning'>[A] bounces off of [src]!</span>")
+		visible_message("<span class='warning'>[A] bounces off [src]!</span>")
```
Comment (ShadowDeath6) [inline code/game/gamemodes/clock_cult/clock_effects/spatial_gateway.dm]: if this uses "the", I'd keep it as it was.

Comment (bgobandit) [inline code/game/gamemodes/clock_cult/clock_effects/spatial_gateway.dm]: "off of" is always redundant

Comment (ShadowDeath6) [inline code/game/gamemodes/clock_cult/clock_effects/spatial_gateway.dm]: "bounces off of the x" sounds better than" bounces off the x", though, even if it is incorrect, imo.

Comment (bgobandit) [inline code/game/gamemodes/clock_cult/clock_effects/spatial_gateway.dm]: ShadowCuck6 sounds better than ShadowDeath6 but it would be incorrect

**Pull request merged**